lib/gitlab/ci/templates/jobs/gitlab/GitlabRelease.yml

```diff
@@ -0,0 +1,19 @@
+gitlab_release:
+  stage: deploy
+  image:
+    name: "gitlab/glab:latest"
+    entrypoint: [""]
+  variables:
+    GITLAB_RELEASE_CLI_FLAGS: 'release create "$CI_COMMIT_TAG" --name "Release ${CI_COMMIT_SHA}" --notes "Created using the gitlab release job." --ref "${CI_COMMIT_SHA}"'
```

Comment:
Probably change "Release ${CI_COMMIT_SHA}" to "Release ${CI_COMMIT_TAG}" to make the Release Page pretty and also the SHA is already on the Release Page so redundant.